scenarios/kubernetes_e2e.py

```diff
@@ -551,7 +551,7 @@ def main(args):
     if args.kubeadm:
         version = kubeadm_version(args.kubeadm, shared_build_gcs_path)
         runner_args.extend([
-            '--kubernetes-anywhere-path=%s' % os.path.join(workspace, 'kubernetes-anywhere'),
+            '--kubernetes-anywhere-path=%s' % os.path.join(workspace, 'k8s.io', 'kubernetes-anywhere'),
```

Comment:
the `--repo=` entries for k-a look like this:
https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm.yaml#L40

here is one of the recent failing logs that shows that the checkout happens in `workspace/k8s.io/kubernetes-anywhere` instead of `workspace/kubernetes-anywhere`
https://storage.googleapis.com/kubernetes-jenkins/logs/ci-kubernetes-e2e-kubeadm-gce-upgrade-1-10-1-11/961/build-log.txt

Reply:
yeah: 
> I0829 21:00:56.980] Checkout: /workspace/k8s.io/kubernetes-anywhere kubeadm-e2e to /workspace/k8s.io/kubernetes-anywhere